docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -138,10 +138,49 @@ To query audio device information:
 
 
         > **Note**   
-		> The initial running state of the connected device is `false`, which means the connected device is not running.
+        > The initial running state of the connected device is `false`, which means the connected device is not running.
```

Comment:
Note tag is still not properly formatted. Note tag should be written with 3 lines at least, each starting with ">". Here there are only two lines. Kindly add the third one also. Note should look like:

>**Note**   
>
>The initial running state of the connected device is `false`, which means the connected device is not running. 

For more information, see: https://github.com/Samsung/tizen-docs/blob/master/styleguide/template-guide.md#note

Reply:
Done. Thanks.

Reply:
Thanks, this change seems fine now.